### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Lucius47**
Minor grammatical changes

### CODE REVIEW

**Review by KendallDoesCoding — CHANGES_REQUESTED** [review]
On line 49 it is: **skeptical out**, I dont think thats english lol and sketched out sounds fine too, so no need of it being skeptical

**Comment by ThioJoe** [comment]
Nah

File: README.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ YouTube offers a functionality to ban a user, but it does NOT delete previous co
 
 ## 🤔 Pro-Tip If This Seems Sketchy: Limiting The App's Access 🤔
 
-If you feel sketched out about giving the app the required high level permissions to your channel (very understandable), you could instead use the app in 'moderator mode' (set in the config file). First, some context: When you grant access to another channel to be a moderator for your channel, they are able to mark comments for 'held for review', and this permission works through the API as well. 
+If you feel skeptical out about giving the app the required high level permissions to your channel (very understandable), you could instead use the app in 'moderator mode' (set in the config file). First, some context: When you grant access to another channel to be a moderator for your channel, they are able to mark comments for 'held for review', and this permission works through the API as well.
```
Comment (Firecul) [inline README.md:49]: > If you feel skeptical out about giving the app[...]

Yeah that's not better

`If you feel skeptical about giving the app [...]`
 would be a better way of saying it

Comment (Lucius47) [reply README.md:49]: Yes, I forgot to take out the "out"

**Pull request closed without merging**